controls/radgridview/events/column-resize-event.md

```diff
@@ -82,51 +82,51 @@ The **GridViewAutoGeneratingColumnEventArgs** class provides you with a few usef
 
 * __Column__: The column that is being reordered.
 
 * __NewDisplayIndex__: The new DisplayIndex of the reordered column.
 
->If ColumnReordering is canceled, the **ColumnDisplayIndexChanged** and __ColumnReordered__ events do not fire. 
+>If you cancel ColumnReordering, **ColumnDisplayIndexChanged** and __ColumnReordered__ events are not triggered. 
 
 ## ColumnDisplayIndexChanged
 
-The __ColumnDisplayIndexChanged__ event occurs when RadGridView's columns have been reordered or  when you have programmatically changed the **DisplayIndex** of a column. It is triggered for every column affected by the reordering (whose DisplayIndex has changed).
+A __ColumnDisplayIndexChanged__ event occurs when you reorder RadGridView's columns or when you programmatically change the **DisplayIndex** of a column. It is triggered for every column affected by the reordering (that is, whose DisplayIndex has changed).
  
-Through the __GridViewColumnEventArgs__ you can access the **Column** whose **DisplayIndex** has changed.
+You can access a **Column** whose **DisplayIndex** has changed using __GridViewColumnEventArgs__.
 
 ## ColumnReordered
 
-The __ColumnReordered__ event occurs after the reorder of columns is complete.
+A __ColumnReordered__ event occurs after the reorder of columns is complete.
 
-Through the __GridViewColumnEventArgs__ you can access the **Column** that has been reordered.
+You can access a **Column** that has been reordered using __GridViewColumnEventArgs__.
 
 ## ColumnWidthChanging
 
-The __ColumnWidthChanging__ event occurs when the column is being resized either programmatically or through the UI.
+A __ColumnWidthChanging__ event occurs when you resize a column programmatically or through the UI.
 
->The user can resize columns only if the __CanUserResizeColumns__ is set to __True__ (which is the default value). 
+>You can resize columns only if __CanUserResizeColumns__ is set to __True__ (which is the default value). 
 
-Through the __ColumnWidthChangingEventArgs__ of its event handler you can access:
+Through the __ColumnWidthChangingEventArgs__ of its event handler you can set the following:
 
-* __Cancel__: By setting this to **True**, you can cancel the resizing of the current column.
+* __Cancel__: Setting this to **True** cancels the resizing of the current column.
 
 * __Column__: The column that is being resized.
 
-* __HorizontalChangeWidth__: A double value which indicates the number of pixels that would be added/subtracted to the column width. If you shrink the width the value will be negative, e.g. -10.
+* __HorizontalChangeWidth__: A double value that indicates the number of pixels that would be added to or subtracted from the column width. If you shrink the width, the value is negative, for example, -10.
 
-* __OriginalWidth__: A double value which represents the original width of the column.
+* __OriginalWidth__: A double value that represents the original width of the column.
```

Comment:
**Regarding Comment 8:** A [double](https://msdn.microsoft.com/en-us/library/678hzkk9.aspx) is a simple numeric type in .NET that stores 64-bit floating-point values. It is already mentioned that this is the number of pixels to be added or removed.